tensorflow/python/keras/regularizers.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, l1=0., l2=0.):  # pylint: disable=redefined-outer-name
     self.l2 = K.cast_to_floatx(l2)
 
   def __call__(self, x):
-    regularization = 0.
+    regularization = ops.convert_to_tensor(0., dtype=K.floatx())
```

Comment:
If l1=0 and l2=0 then we would be creating a useless node in the graph. So please place this inside a `if l1 or l2:`. Then please add some unit tests.